### ISSUE TITLE
Accessibility: Improve accessibility of mybinder about supporters page.

### ISSUE THREAD

**Issue by Kemmie001** [jupyterhub/outreachy#63]
### What page is this for?

https://mybinder.readthedocs.io/en/latest/about/supporters.html

### WAVE accessibility report

1. The logo image doesn't have a very descriptive alternative text
2. The cloud credit supporters image on this page has no alternative text
3. Empty links without text close to the cloud credit supporters' images
4. Some areas have low color contrast for text and background.
![Screenshot 2022-10-16 at 10-41-29 WAVE Report of Supporting organizations — Binder 0 1b documentation](https://user-images.githubusercontent.com/60070132/196029161-700bdae5-a201-40de-9853-15ac0bb13157.png)

### The HTML element to be changed

```html
1. Suspicious alternative text i.e the logo image without descriptive alternative text
<a class="navbar-brand logo" href="../index.html">
Suspicious alternative text<img src="../_static/logo.png" class="logo__image only-light" alt="Logo image">
<img src="../_static/logo.png" class="logo__image only-dark" alt="Logo image"></a>

2. The cloud credit supporter's image
   i. <img src="https://www.gesis.org/typo3conf/ext/gesis_web_ext/Resources/Public/webpack/dist/img  /gs_home_logo_de.svg" style="max-height:5em;min-height:2em;">
  ii. <img src="https://www.gstatic.com/devrel-devsite/prod/v6cd15f45ec209c8961e07ea7e57ed9a0e9da4333bc915e67d1fcd2b2a9ec62d1/opensource/images/lockup.svg" style="max-height:5em;min-height:2em;">
   iii. <img src="https://upload.wikimedia.org/wikipedia/commons/2/26/Logo-OVH.svg" style="max-height:5em;min-height:2em;">
   iv. <img src="https://upload.wikimedia.org/wikipedia/commons/b/b5/Alan_Turing_Institute_logo.svg" style="max-height:5em;min-height:2em;">

3. Empty links without text
   i <a class="sd-stretched-link reference external" href="https://www.gesis.org"></a>
   ii <a class="sd-stretched-link reference external" href="https://opensource.google/"></a>
   iii <a class="sd-stretched-link reference external" href="https://www.ovhcloud.com/en/"></a>
   iv <a class="sd-stretched-link reference external" href="https://www.turing.ac.uk/"></a>

4. Areas with low color contrast
   i <a rel="nofollow noopener" target="_blank" href="https://ethicalads.io?ref=rtd-sidebar&ea-publisher=readthedocs">
Ad by EthicalAds
</a>
  ii <a rel="nofollow noopener" target="_blank" href="https://www.ethicalads.io/publishers/?ref=rtd-sidebar-join&ea-publisher=readthedocs">
Host these ads
</a>
  iii <strong>GESIS</strong>
  iv  <strong>Google Open Source</strong>
  v <strong>OVHCloud</strong>
  vi <strong>The Turing Institute</strong>
```

### Your proposed HTML (or CSS), after change

```html
1. <a class="navbar-brand logo" href="../index.html">
Suspicious alternative text<img src="../_static/logo.png" class="logo__image only-light" alt="Binder user guide logo image">
<img src="../_static/logo.png" class="logo__image only-dark" alt="Binder user guide logo image"></a>

2. 
  i.  <img src="https://www.gesis.org/typo3conf/ext/gesis_web_ext/Resources/Public/webpack/dist/img  /gs_home_logo_de.svg" style="max-height:5em;min-height:2em;" alt="Gesis logo image">

  ii. <img src="https://www.gstatic.com/devrel-devsite/prod/v6cd15f45ec209c8961e07ea7e57ed9a0e9da4333bc915e67d1fcd2b2a9ec62d1/opensource/images/lockup.svg" style="max-height:5em;min-height:2em;" alt="Google open source logo image">
  
  iii. <img src="https://upload.wikimedia.org/wikipedia/commons/2/26/Logo-OVH.svg" style="max-height:5em;min-height:2em;" alt="OVHCloud logo image">

  iv. <img src="https://upload.wikimedia.org/wikipedia/commons/b/b5/Alan_Turing_Institute_logo.svg" style="max-height:5em;min-height:2em;" alt="The turning institute logo image">

3. 
// Put the image inside the anchor tag
   i <a class="sd-stretched-link reference external" href="https://www.gesis.org">
<img src="https://www.gesis.org/typo3conf/ext/gesis_web_ext/Resources/Public/webpack/dist/img  /gs_home_logo_de.svg" style="max-height:5em;min-height:2em;" alt="Gesis logo image">
</a>
   ii <a class="sd-stretched-link reference external" href="https://opensource.google/">
<img src="https://upload.wikimedia.org/wikipedia/commons/b/b5/Alan_Turing_Institute_logo.svg" style="max-height:5em;min-height:2em;" alt="The turning institute logo image">
</a>
   iii <a class="sd-stretched-link reference external" href="https://www.ovhcloud.com/en/">
<img src="https://upload.wikimedia.org/wikipedia/commons/2/26/Logo-OVH.svg" style="max-height:5em;min-height:2em;" alt="OVHCloud logo image">
</a>
   iv <a class="sd-stretched-link reference external" href="https://www.turing.ac.uk/">
<img src="https://upload.wikimedia.org/wikipedia/commons/b/b5/Alan_Turing_Institute_logo.svg" style="max-height:5em;min-height:2em;" alt="The turning institute logo image">
</a>

4. 
  i <strong style="color: #000000;">GESIS</strong>
  ii  <strong style="color: #000000;">Google Open Source</strong>
  iii <strong style="color: #000000;">OVHCloud</strong>
  iv <strong style="color: #000000;">The Turing Institute</strong>
```

**Comment by minrk** [comment]
Good suggestions! We'd need to futz with the CSS to get the alignment right after moving it, but putting the img in the anchor should be better.

**Comment by Kemmie001** [comment]
@minrk Yes would you like me to write out the changes that the CSS will need out also

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
